src/estimate.coffee

```diff
@@ -220,3 +291,32 @@ module.exports = (robot) ->
         "#{NAMESPACE}#{ticketId}#{TOTAL_VOTERS}", { room, votersCount }
       )
       res.send "Waiting for #{votersCount} votes to print max for #{ticketId}"
+
+  robot.respond /blocks test/i, id: 'blocks.test', (res) ->
+    args = {
+      token: robot.adapter.options.token,
+      channel: res.message.room,
+      as_user: true,
+      blocks: blocks()
+    }
+    slackRequest(robot, "chat.postMessage").query(args).get() (err, _, body) ->
+      console.log("err: #{err}, body: #{body}")
+
+  if robot.router.use
+    robot.router.use(
+      process.env.SLACK_ACTIONS_URL || "/slack/actions", messageMiddleware
+    )
+    robot.router.use(
+      process.env.SLACK_OPTIONS_URL || "/slack/options", messageMiddleware
+    )
+
+    middleware = robot.router._router.stack.slice()
```

Comment:
See https://github.com/slackapi/node-slack-interactive-messages/issues/86#issuecomment-487176798 for an explanation of why this is necessary